--- final-vote-archive.txt	Thu Nov  9 04:40:11 2000
+++ final-vote-archive.fixed	Thu Nov  9 04:09:55 2000
@@ -14789,13 +14789,13 @@
 Precedence: bulk
 List-Id:  <vote.gnome.org>
 
-cm@ggtt.de
-e6c96d3c4599b718988da049f4c12154
+Member Address: cm@ggtt.de
+Validation Token: e6c96d3c4599b718988da049f4c12154
 
 MARTIN BAULIG (ID# 1)
 KENNETH CHRISTIANSEN (ID# 3)
 KJARTAN MARAAS (ID# 19)
-AN MUETH (ID# 21)
+DAN MUETH (ID# 21)
 HAVOC PENNINGTON (ID# 23)
 TUOMAS KUOSMANEN (ID# 15)
 CHRISTOPHER GABRIEL (ID# 7)
@@ -14872,19 +14872,19 @@
 Precedence: bulk
 List-Id:  <vote.gnome.org>
 
-baudais@okstate.edu
-5c46dd42d39e2445d872bd4a90c627ed
-MACIEJ STACHOWIAK 31
-ARLO ROSE 29
-DAN MUETH 21
-KJARTAN MARAAS 19
-MARTIN BAULIG 1
-JIM GETTYS 9
-JOHN HEARD 12
-BART DECREM 5
-BRUCE PERENS 25
-DANIEL VEILLARD 33
-CHEMA CELORIO 2
+Member Address: baudais@okstate.edu
+Validation Token: 5c46dd42d39e2445d872bd4a90c627ed
+MACIEJ STACHOWIAK (ID# 31)
+ARLO ROSE (ID# 29)
+DAN MUETH (ID# 21)
+KJARTAN MARAAS (ID# 19)
+MARTIN BAULIG (ID# 1)
+JIM GETTYS (ID# 9)
+JOHN HEARD (ID# 12)
+BART DECREM (ID# 5)
+BRUCE PERENS (ID# 25)
+DANIEL VEILLARD (ID# 33)
+CHEMA CELORIO (ID# 2)
 
 
 
@@ -15874,7 +15874,7 @@
 BERTRAND GUIHENEUF (ID# 10)
 JOHN HEARD (ID# 12)
 ELLIOT LEE (ID# 17)
-RAPH LEVIAN (ID# 18)
+RAPH LEVIEN (ID# 18)
 LESLIE PROCTOR (ID# 26)
 ARLO ROSE (ID# 29)
 OWEN TAYLOR (ID# 32)
@@ -15901,29 +15901,5 @@
 -----END PGP SIGNATURE-----
 
 --Dxnq1zWXvFF0Q93v--
-
-
-From martin@cvs.gnome.org  Thu Nov  9 04:21:56 2000
-Return-Path: <martin@cvs.gnome.org>
-Delivered-To: vote@gnome.org
-Received: from cvs.gnome.org (canvas.gnome.org [209.116.70.70])
-	by mail.gnome.org (Postfix) with ESMTP id DABD12BDAF
-	for <vote@gnome.org>; Thu,  9 Nov 2000 04:21:56 -0500 (EST)
-Received: by cvs.gnome.org (Postfix, from userid 2159)
-	id 2C8013360B7; Thu,  9 Nov 2000 04:21:55 -0500 (EST)
-To: vote@gnome.org
-Subject: Closed
-Message-Id: <20001109092155.2C8013360B7@cvs.gnome.org>
-Date: Thu,  9 Nov 2000 04:21:55 -0500 (EST)
-From: martin@cvs.gnome.org (Martin Baulig (martin@home-of-linux.org))
-Sender: vote-admin@gnome.org
-Errors-To: vote-admin@gnome.org
-X-BeenThere: vote@gnome.org
-X-Loop: vote@gnome.org
-X-Mailman-Version: 2.0beta5
-Precedence: bulk
-List-Id:  <vote.gnome.org>
-
-Thu Nov  9 04:21:55 EST 2000
 
 
